api/src/org/apache/cloudstack/api/command/user/snapshot/RevertSnapshotCmd.java

```diff
@@ -31,25 +31,36 @@
 import org.apache.cloudstack.api.response.SnapshotResponse;
 import org.apache.cloudstack.api.response.SuccessResponse;
 import org.apache.cloudstack.context.CallContext;
+import org.apache.log4j.Logger;
 
 import com.cloud.event.EventTypes;
 import com.cloud.storage.Snapshot;
 import com.cloud.user.Account;
 
-@APICommand(name = "revertSnapshot", description = "revert a volume snapshot.", responseObject = SnapshotResponse.class, entityType = {Snapshot.class},
+@APICommand(name = "revertSnapshot", description = "revert a volume snapshot.", responseObject = SuccessResponse.class, entityType = {Snapshot.class},
```

Comment:
I think for API backward compatibility, please revert this to send SnapshotResponse as the responseobject, unless this really needs to be fixed (the older implementation got it wrong). Please comment.